### PR TITLE
ci: Add new step on CI to scan for secrets

### PR DESCRIPTION
**Pull request by w0rmr1d3r**
Adding Gitleaks into our CI.

Using it as a docker image, since the GitHub action doesn't say how to put more options.

### CODE REVIEW

**Review by edufarre — COMMENTED** [review]
I would change the title of the PR (commit message when merging) to use the `semantic release` commit messages:
```
ci: Changes to our CI configuration files and scripts (example scopes: Circle, BrowserStack, SauceLabs)
```
thus having something like: `ci: Add new step on CI to scan for secrets`

https://github.com/angular/angular/blob/master/CONTRIBUTING.md#type

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,6 +19,9 @@ jobs:
         with:
           fetch-depth: 0 # 👈 Required to retrieve git history
 
+      - name: Gitleaks # Can we have all these options in the Gitleaks GitHub action? -> https://github.com/zricethezav/gitleaks-action/issues/36
+        run: docker run -v ${PWD}:/my-repo zricethezav/gitleaks:latest --path="/my-repo" --no-git --verbose
```
Comment (edufarre) [inline .github/workflows/ci.yml:23]: Why docker and not just running the existing GA? I've seen you opened an issue in their repo about trying to add the `--no-git` and `--verbose`, but did you try just running it and seeing which are the logs output on Github actions?

**Comment by w0rmr1d3r** [comment]
done @edufarre !

**Review by edufarre — APPROVED** [review]

**Pull request merged**